### PR TITLE
ON-12266 # Implemented refactored form schema validation

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/lib/forms-schema/elements/HeadingElement.ts [hunk 1]
```diff
@@ -10,4 +10,4 @@ export default Joi.object({
     .required()
     .label('Form Element - Heading Size')
     .valid([1, 2, 3, 4, 5]),
-})
```
Comment (mymattcarroll) [inline src/lib/forms-schema/elements/HeadingElement.ts:13]: Why was this added here and not to the others?

Comment (Zaxist) [reply src/lib/forms-schema/elements/HeadingElement.ts:13]: Because there was a test using this element, when I was trying to figure out where this needed to go. Forgot to remove
